### PR TITLE
Create ui-c8y-1019-7-11-fix-navigator-app-icon-size-var-5756-graft-release-cd-5769.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-57153](https://cumulocity.atlassian.net/browse/MTM-57153)
Original PR: [5769](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5769)

### CODE REVIEW

File: content/change-logs/ui-c8y-1019-7-11-fix-navigator-app-icon-size-var-5756-graft-release-cd-5769.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fix navigator app icon size var (#5756) [GRAFT][release/cd] (#5769)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1019-7-11-fix-navigator-app-icon-size-var-5756-graft-release-cd-5769.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fix app icon size variable in navigator
```

File: content/change-logs/ui-c8y-1019-7-11-fix-navigator-app-icon-size-var-5756-graft-release-cd-5769.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57153
+version: 1019.7.11
+---
+Fix navigator app icon size var (#5756) [GRAFT][release/cd] (#5769)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1019-7-11-fix-navigator-app-icon-size-var-5756-graft-release-cd-5769.md:17]: The Claude AI suggested to use the following body:
```suggestion
Fixed an issue with the variable that controls the size of app icons displayed in the navigator. This corrects icon sizes that were incorrectly scaled in certain cases.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by janhommes — APPROVED** [review]

File: content/change-logs/ui-c8y-1019-7-11-fix-navigator-app-icon-size-var-5756-graft-release-cd-5769.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fix css-variable for sizing the app icon
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-7-11-fix-navigator-app-icon-size-var-5756-graft-release-cd-5769.md:3]:
```suggestion
title: Css variable now correctly sizes the app icon
```

Comment (BeateRixen) [reply content/change-logs/ui-c8y-1019-7-11-fix-navigator-app-icon-size-var-5756-graft-release-cd-5769.md:3]: @janhommes Is this rephrasing correct? We should not start the titles with "Fixed" but better clearly state what works now.

Comment (janhommes) [reply content/change-logs/ui-c8y-1019-7-11-fix-navigator-app-icon-size-var-5756-graft-release-cd-5769.md]: yes, sounds good.

File: content/change-logs/ui-c8y-1019-7-11-fix-navigator-app-icon-size-var-5756-graft-release-cd-5769.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Css variable now correctly sizes the app icon
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-7-11-fix-navigator-app-icon-size-var-5756-graft-release-cd-5769.md:3]:
```suggestion
title: CSS variable now correctly sizes the app icon
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**